charts/tidb-cluster/values.yaml

```diff
@@ -359,7 +359,9 @@ monitor:
   storage: 10Gi
   initializer:
     image: pingcap/tidb-monitor-initializer:v3.0.1
-    imagePullPolicy: IfNotPresent
+    imagePullPolicy: Always
```

Comment:
why change the `imagePullPolicy` to `Always`?

Reply:
The new tidb operator is incompatible with old image and for now the monitoring image changes frequency, so we think it is better to update the policy with `Always`